src/lambda/commands/invokeLambda.ts

```diff
@@ -195,3 +181,12 @@ function createMessageReceivedFunc({
         }
     }
 }
+
+function makeSampleRequestManifestResourceFetcher(): ResourceFetcher {
+    const resourcePath = ext.context.asAbsolutePath(path.join('resources', 'vs-lambda-sample-request-manifest.xml'))
```

Comment:
Can we throw this into constants as well (a la `sampleRequestManifestPath`)?

Reply:
Let me see what I can do

Reply:
I'm also fine with moving the `sampleRequestManifestPath` to this function if it isn't used elsewhere. It's just jarring to see the two in different places (same for the other one below)